src/packages/recompose/createEventHandler.js

```diff
@@ -0,0 +1,23 @@
+import $$observable from 'symbol-observable'
+import { createChangeEmitter } from 'change-emitter'
+import { getTransform } from './configureObservable'
+
+const createEventHandler = () => {
+  const emitter = createChangeEmitter()
+  const transform = getTransform()
```

Comment:
@acdlite Maybe allow pass to `configureObservable` subject-like anything? And use `change-emitter` if only passed `Observable` dont has method `next`?